python/sglang/bench_latency.py

```diff
@@ -218,15 +234,25 @@ def latency_test(
         f"max_batch_size={model_runner.max_total_num_tokens // (bench_args.input_len + bench_args.output_len)}"
     )
 
+    # To make this PR easier to review, for now, only do the first element in batch_size tuple.
+    bench_args.batch_size = bench_args.batch_size[0]
```

Comment:
Remove this line or do you want to do it in the follow up PR?

Reply:
That's my thought to make the review easier. If you prefer, I can include in this PR another commit for the part 2 as well. For part 2, I plan to move the nested function out and call them in a loop.

Reply:
please unresolve this if you like to combine step 1 and step 2 in this PR.